pkg/cloud/aws/executor.go

```diff
@@ -61,7 +61,7 @@ func (b BackoffExecutor) Execute(ctx context.Context, f RequestFunction) (interf
 		req.SetContext(ctx)
 		err := req.Send()
```

Comment:
shouldn't we actually check this first before checking the reponses status code?

Reply:
I... do exactly this with my change set? If the error is not nil, the status code is not checked and I just propagate the error?